regolith/tools.py

```diff
@@ -1497,21 +1497,21 @@ def collect_appts(ppl_coll, filter_key=None, filter_value=None, begin_date=None,
                 appts[-1].update({'person': p.get('_id')})
     return appts
 
-def get_grant_amount(grant, ppl_coll, begin_date=None, end_date=None):
+def grant_burn(grant, appts, begin_date=None, end_date=None):
     """
-    Given a grant, a database of people appointed on assorted grants, and two dates marking the limits of an interval,
-    it returns a list of the net amount of the grant on each day of the interval.
+    Retrieves the total burn of a grantt over an interval of time by integrating over all appointments
+    made on the grant.
 
     Parameters
     ----------
     grant: dict
-        The person whose appointments need to be checked
-    ppl_coll: collection (list of dicts)
-        The people collection that contains the grant appointments
+        The grant_id or grant object whose burn needs to be retrieved
```

Comment:
this doesn't seem to be consistent.  It says type dict but then you say grant_id is ok

Reply:
removed the `grant_id` option! I thought we could allow both and use `client.find_one` to retrieve the grant, but I'm not sure that would work nested in the function since the grants database varies